### PR TITLE
feat: support K8S application logs

### PR DESCRIPTION
**Pull request by harelmo-lumigo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -73,7 +73,7 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
-	(cd ./controller/src && KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" $(GOCMD) test . -coverprofile cover.out )
+	(cd ./controller/src && KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" $(GOCMD) test ./... -failfast -coverprofile cover.out )
```
Comment (harelmo-lumigo) [inline Makefile:76]:
```suggestion
	(cd ./controller/src && KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" $(GOCMD) test ./... -coverprofile cover.out )
```

File: telemetryproxy/src/processor/k8sdataenricherprocessor/processor.go [hunk 2]
```diff
@@ -72,122 +72,127 @@ func (kp *kubernetesprocessor) processTraces(ctx context.Context, tr ptrace.Trac
 	resourceSpanLength := resourceSpans.Len()
 	for i := 0; i < resourceSpanLength; i++ {
 		resource := resourceSpans.At(i).Resource()
-		resourceAttributes := resource.Attributes()
 
-		resourceAttributes.PutStr(K8SProviderIdKey, kp.kube.GetProviderId())
-		resourceAttributes.PutStr(K8SClusterUIDKey, string(kp.clusterUid))
+		kp.addResourceAttributes(ctx, resource)
```
[the remark this answers is not in the document]
Comment (harelmo-lumigo) [reply telemetryproxy/src/processor/k8sdataenricherprocessor/processor.go:76]: I've extracted the logic to add attributes to be shared to both spans and logs (currently it was hard-coded into `processSpans`

File: charts/lumigo-operator/values.yaml [hunk 3]
```diff
@@ -61,4 +61,5 @@ metricsService:
   type: ClusterIP
 endpoint:
   otlp:
-    url: https://ga-otlp.lumigo-tracer-edge.golumigo.com
\ No newline at end of file
+    url: https://ga-otlp.lumigo-tracer-edge.golumigo.com
+    logs_url: https://ga-otlp.lumigo-tracer-edge.golumigo.com
```
[the remark this answers is not in the document]
Comment (harelmo-lumigo) [reply charts/lumigo-operator/values.yaml:65]: For flexibility - we can always reduce it down to a single env-var once we're 100% sure it will be the same

**Review by moshe-shaham-lumigo — DISMISSED** [review]

File: charts/lumigo-operator/values.yaml [hunk 4]
```diff
@@ -61,4 +61,5 @@ metricsService:
   type: ClusterIP
 endpoint:
   otlp:
-    url: https://ga-otlp.lumigo-tracer-edge.golumigo.com
\ No newline at end of file
+    url: https://ga-otlp.lumigo-tracer-edge.golumigo.com
```
Comment (lir-lumigo) [inline charts/lumigo-operator/values.yaml:64]: @harelmo-lumigo `url` will become the tracing endpoint url?
If so, can we change the name to `traces_url`? (and keep `url` for backward compatibility)

**Review by moshe-shaham-lumigo — APPROVED** [review]

**Pull request merged**